students/jemielity_kamil/lesson_06_dicts_tuples_sets_args_kwargs/frequency_analysis.py

```diff
@@ -0,0 +1,24 @@
+
+Dictionary = {}
+for i in range(int(input('How many lines of text: '))):
+    sentence = input('Sentence: ').split()
+    for sentence_word in sentence:
+        if sentence_word not in Dictionary.keys():
+            Dictionary[sentence_word] = 1
+        else:
+            Dictionary[sentence_word] += 1
+
+Dictionary = list(Dictionary.items())
```

Comment:
It is not a good practice to change variable type

dict {} -> list []

Please create separate variable for list